script.js

```diff
@@ -0,0 +1,403 @@
+console.log(getBillboardPrice('To be, or not to be', 20)); // 380
+
+// 9.Very simple, given a number (integer / decimal / both depending on the language), find its opposite (additive inverse).
+
+function getOppositeNumber(number) {
+    return number * -1;
+}
+
+console.log(getOppositeNumber(-0.75));
+
+// 10. In this simple assignment you are given a number and have to make it negative.
+// But maybe the number is already negative?
+
+function makeNegative(number) {
+    if (number > 0) {
+        return number * -1;
+    }
+}
+
+console.log(makeNegative(5));
+
+// 11. It's pretty straightforward.
+// Your goal is to create a function that removes the first and last characters of a string.
+// You're given one parameter, the original string. You don't have to worry about strings with less than two characters.
+
+function firstAndLastLetter(word) {
+    return word.slice(1,-1);
+}
+
+console.log(firstAndLastLetter("precisely"));
+
+// 12. Your task is to create a function that does four basic mathematical operations.
+// The function should take three arguments - operation(string/char), value1(number), value2(number).
+// The function should return result of numbers after applying the chosen operation.
+
+function basicMathOperation(operation, firstValue, secondValue) {
+    if (operation === '+') {
+        return firstValue + secondValue;
+    }
+    if (operation === '-') {
+        return firstValue - secondValue;
+    }
+    if (operation === '*') {
+        return firstValue * secondValue;
+    }
+    if (operation === '/') {
+        return firstValue / secondValue;
+    }
+}
+
+console.log(basicMathOperation('*', 10, 2));
+
+// 13. The first century spans from the year 1 up to and including the year 100,
+// the second century - from the year 101 up to and including the year 200, etc.
+// Task
+// Given a year, return the century it is in.
+
+function getCentury(year) {
+    return Math.ceil(year / 100);
+}
+
+console.log(getCentury(1758));
+
+// 14. Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
+
+function isEvenOrOdd(number) {
+    if (number % 2 === 0) {
+        return 'Even';
+    }
+    return 'Odd';
+}
+
+console.log(isEvenOrOdd(53));
+
+// 15. Create a function which answers the question "Are you playing banjo?".
+// If your name starts with the letter "R" or lower case "r", you are playing banjo!
+
+function areYouPlayingBanjo(name) {
+    const firstLetter = name.charAt(0);
+    if (firstLetter === 'R' || firstLetter === 'r') {
+        return name + ' plays banjo!';
+    }
+    return name + ' does not play banjo!';
+}
+
+console.log(areYouPlayingBanjo("Robert"));
+
+// 16. Write function bmi that calculates body mass index (bmi = weight / height2).
+//
+// if bmi <= 18.5 return "Underweight"
+// if bmi <= 25.0 return "Normal"
+// if bmi <= 30.0 return "Overweight"
+// if bmi > 30 return "Obese"
+
+function BMI(weight, height) {
```

Comment:
Figure out a better name than `BMI`. I suggest something like `getBMI` because we are getting something here based on the weight and height